### PR TITLE
Add sign apk code

### PR DESCRIPTION
**Pull request by zebra-lucky**
* add sign apk code to `contrib/sign-releases.py`
* remove unused `check_github_repo`
* change version to 3.0.6.1, add RELEASE-NOTES

### CODE REVIEW

File: contrib/sign-releases.py [hunk 1]
```diff
@@ -2,10 +2,58 @@
 # -*- coding: utf-8 -*-
 """Sign releases on github, make/upload ppa to launchpad.net
 
-NOTE: To build ppa need to install some packages. On ubuntu:
+NOTE on ppa: To build ppa need to install some packages. On ubuntu:
```
Comment (jardev) [inline contrib/sign-releases.py]: Language issue. Let's better write like following: To build a ppa you may need to install some more packages. On ubuntu:

File: contrib/sign-releases.py [hunk 2]
```diff
@@ -2,10 +2,58 @@
...
+
      sudo apt-get install devscripts libssl-dev python3-dev \
           debhelper python3-setuptools dh-python
 
+NOTE on apk signing: To create keystore and sign apk need to install
```
Comment (jardev) [inline contrib/sign-releases.py]: Same thing and don't forget about articles. Let's rewrite it as "To create a keystore and sign the apk you need to install"?

File: contrib/sign-releases.py [hunk 3]
```diff
@@ -2,10 +2,58 @@
...
+      java-8-openjdk, or java-7-openjdk on older systems.
+
+To create keystore run following command:
```
Comment (jardev) [inline contrib/sign-releases.py]: -> "To create a keystore run the following command:"

File: contrib/sign-releases.py [hunk 4]
```diff
@@ -2,10 +2,58 @@
...
+
+    mkdir ~/.jks && keytool -genkey -v -keystore ~/.jks/keystore \
+        -alias electrum.dash.org -keyalg RSA -keysize 2048 \
+        -validity 10000
+
+Then it's show warning about proprietary format and command to migrate:
```
Comment (jardev) [inline contrib/sign-releases.py]: Then it shows a warning about the proprietary format and a command to migrate:

File: contrib/sign-releases.py [hunk 5]
```diff
@@ -2,10 +2,58 @@
...
+
+    jarsigner -verbose \
+        -tsa http://sha256timestamp.ws.symantec.com/sha256/timestamp \
+        -sigalg SHA1withRSA -digestalg SHA1 \
+        -sigfile dash-electrum \
+        -keystore ~/.jks/keystore \
+        Electrum_DASH-3.0.6.1-release-unsigned.apk \
+        electrum.dash.org
+
+Zipalign from Android SDK build tools is also need(set path to bin in
```
Comment (jardev) [inline contrib/sign-releases.py]: Zipalign from Android SDK build tools is also required (set path...

File: contrib/sign-releases.py [hunk 6]
```diff
@@ -325,6 +404,60 @@ def __init__(self, **kwargs):
             while not self.passphrase:
                 self.read_passphrase()
 
+        if self.zipalign_path:
+            try:
+                check_call(self.zipalign_path, stderr=FNULL)
+            except CalledProcessError:
+                pass
```
Comment (jardev) [inline contrib/sign-releases.py]: Shouldn't we log it when calling zipalign fails?

Comment (zebra-lucky) [reply contrib/sign-releases.py]: This is to check if `self.zipalign_path` is can launch, and without arguments it's raises CalledProcessError.

**Pull request merged**